src/typeahead/typeahead.directive.ts

```diff
@@ -119,6 +119,8 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
 
   /** This attribute indicates that the dropdown should be opened upwards */
   @Input() dropup = false;
+  /** if true, typeahead will cancel async request on blur */
+  @Input() typeaheadCancelOnBlur = false;
```

Comment:
this option will not be changed too often, please move to it to config,
to avoid Component bloating